lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1649,6 +1667,15 @@ def _get_expected_ulimits(self, config_ulimits):
             ))
         return results
 
+    def _get_expected_sysctls(self, config_sysctls):
+        self.log('_get_expected_sysctls')
+        if config_sysctls is None:
+            return None
+        result = dict()
```

Comment:
My bad, I forgot that [dict comprehensions were added in Python 2.7](https://www.python.org/dev/peps/pep-0274/).  Feel free to ignore my suggestion.